packages/api-nest/src/queue/processors/content-processor.service.ts

```
@@ -355,6 +381,25 @@ export class ContentProcessorService
     }
   }
 
+  /**
+   * Generate SHA-256 hash of content for duplicate detection
+   *
+   * @param content - Content to hash
+   * @returns SHA-256 hash as hex string
+   */
```

ESLint

[error] Missing JSDoc return type.

(valid-jsdoc)